extensions/blocks/contact-form/editor.scss

```diff
@@ -3,7 +3,7 @@
 .wp-block-jetpack-contact-form {
 	.block-editor-block-variation-picker__variations > li {
```

Comment:
I'm thinking we may need to tweak the labels below each variation. Now that we have less room per variation, some bad wrapping may happen:

![image](https://user-images.githubusercontent.com/426388/90251896-d6204880-de3e-11ea-9b7a-4a3fa91f8e96.png)

Reply:
This looks better? I noticed that the text was not in the middle of the box. once I have put the labels in the middle. It looks fine. 
![image](https://user-images.githubusercontent.com/47211808/90258384-c1e14900-de48-11ea-933e-a2ef00916319.png)